tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrdersControllerTests.cs

```diff
@@ -308,6 +309,46 @@ public async Task GetOrderSummaryAsync_CatalogueSolutionCount_ReturnsCountOfTwo(
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [TestCase]
+        public async Task GetOrderSummaryAsync_AssociatedServiceCount_ReturnsCountOfTwo()
```

Comment:
A suggestion: not required, but it'd be good to add a non associated service into the list to ensure it's filtering

Reply:
👍 good suggestion. Will add it in